sample/HttpTrigger-TypeScript/index.ts

```diff
@@ -0,0 +1,29 @@
+﻿export function index(context: any, req: any) {
```

Comment:
we'll need to add better type info for context and bindings - though probably out of scope for this PR.